cluster/manifests/flannel/daemonset.yaml

```diff
@@ -66,15 +66,8 @@ spec:
         effect: NoExecute
       - operator: Exists
         key: CriticalAddonsOnly
-      affinity:
-        nodeAffinity:
-          requiredDuringSchedulingIgnoredDuringExecution:
-            nodeSelectorTerms:
-            - matchExpressions:
-              - key: flannel
-                operator: NotIn
-                values:
-                - local
+      - key: node-role.kubernetes.io/master
+        effect: NoSchedule
```

Comment:
I believe this toleration must be applied before rolling the nodes.

Reply:
No, scratch that. Normally yes but the Pod also has a "tolerate everything" tolerations below. This is therefore not doing anything and was just added to be similar with the other DaemonSets.